src/pages/questionPage.js

```diff
@@ -7,6 +7,9 @@ import { createQuestionElement } from '../views/questionView.js';
 import { createAnswerElement } from '../views/answerView.js';
 import { quizData } from '../data.js';
 
+// Counts how many answers are correct. Every right answer will increment this variable by 1
+let correctCounter = 0;
```

Comment:
I don't think keeping count was part of this task. It's best to tackle one problem at a time.

Reply:
deleted all that related to displaying scores. will do this later